modules/components/pages/inputs/redpanda_common.adoc

```diff
@@ -0,0 +1,290 @@
+= redpanda_common
+// tag::single-source[]
+:type: input
+:page-beta: true
+:categories: ["Services"]
+
+component_type_dropdown::[]
+
+
+Consumes data from a Redpanda (Kafka) broker, using credentials from a common `redpanda` configuration block.
+
+ifndef::env-cloud[]
+Introduced in version 4.39.0.
+endif::[]
+
+Use this connector in conjunction with the xref:components:outputs/redpanda_common.adoc[`redpanda_common` output], to read data from one or more topics on a Redpanda cluster, and then write the data to another topic on the same cluster.
```

Comment:
Is this correct? Just trying to make it easier for users to understand when/if to use this component.

Reply:
Addressed above

Reply:
Removed.